### PR TITLE
Add CC-BY-SA-2.0-UK license and test

### LINKED ISSUE #1106: New license request: CC-BY-SA-2.0-UK [SPDX-Online-Tools]
**1.** License Name: Creative Commons Attribution Share Alike 2.0 England and Wales
**2.** Short identifier: CC-BY-SA-2.0-UK
**3.** License Author or steward: 
**4.** Comments: 
**5.** Standard License Header: There is no standard license header for the license
**6.** URL: https://creativecommons.org/licenses/by-sa/2.0/uk/legalcode
**7.** OSI Status: Not Submitted

### PR DESCRIPTION
**Pull request by swinslow**
Fixes #1106 

Signed-off-by: Steve Winslow <steve@swinslow.net>

### CODE REVIEW

**Comment by swinslow** [comment]
@jlovejoy @kestewart @rpurdie or others, if someone can be a second pair of eyes and give this a once-over, we can merge it. Thanks!

**Comment by rpurdie** [comment]
I am no SPDX expert but I did quickly scan the commit and it looks reasonable to me as far as I can tell, thanks!

**Review by kestewart — APPROVED** [review]
Some of the spacing looks a bit weird to me, but I assume that is how it is in the original and we're just preserving it.

**Pull request merged**
